integration/sawtooth_integration/tests/test_digest_auth_proxy.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+# pylint: disable=protected-access
```

Comment:
Is this necessary for this file? Looks like the only protected access is for the test's own `_assert_valid_authed_request`, which shouldn't trip the linter as I understand it.

Reply:
This should be removed (we shouldn't ignore lints), and the underscore could be removed.